demos/connector/neo4j/undirected-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -0,0 +1,1117 @@
+{
```

Comment:
<p>Maybe the title should say something about this being an example using Neo4j? For example:</p><pre class="ql-syntax" spellcheck="false">GraphSAGE on the CORA citation network using StellarGraph and Neo4j
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>